frontend/components/Chart/style.scss

```
@@ -1,6 +1,27 @@
 @use '@/static/styles/package' as *;
 #invoice-chart {
   height: auto;
+  #chart-selector {
+    margin: 0 auto;
+    width: 30%;
+  }
+  #bar-chart {
+    position: relative;
+  }
+  #bar-display {
+    position: absolute;
+    background-color: black;
+    color: white;
```

```suggestion
    background-color: $color-black;
    color: $color-white;
```
이왕이면 이미 선언된 scss 변수를 사용하는 게 좋을 것 같습니다!